setup.py

```diff
@@ -89,7 +89,7 @@ def _global_version(version):
     keywords="ethereum evm smart contract language",
     include_package_data=True,
     packages=find_packages(exclude=("tests", "docs")),
-    python_requires=">=3.7,<3.11",
+    python_requires=">=3.8,<4",
```

Comment:
Not sure when we can support 3.12, but probably not out of the gate
```suggestion
    python_requires=">=3.8,<3.12",
```

Reply:
i think we should leave the upper bound unspecified so if/when the next python is released, people can try using vyper without us needing to explicitly open a PR and testing it

Reply:
Okay. I suppose the metadata shows that we don't explicitly support it yet